### PR TITLE
Add v3.0-RC2 support

### PR DESCRIPTION
**Pull request by richfab**
This PR adds v3.0-RC2 support in the GBFS validator.

I chose to replace the v3.0-RC tests with v3.0-RC2 tests as the v3.0-RC2 changes includes the v3.0-RC changes.

A thorough review would be appreciated as this is my first version update contribution to the validator. Thank you!

### CODE REVIEW

**Comment by tdelmas** [comment]
LGTM

File: gbfs-validator/versions/partials/v3.0-RC2/system_information/required_store_uri.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+module.exports = ({ android = false, ios = false }) => {
+  const r = {
```
Comment (testower) [inline gbfs-validator/versions/partials/v3.0-RC2/system_information/required_store_uri.js:2]: What does "r" mean?

File: gbfs-validator/versions/partials/v3.0-RC2/vehicle_status/required_vehicle_type_id.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+module.exports = ({ vehicleTypes }) => {
+  const res = {
```
Comment (testower) [inline gbfs-validator/versions/partials/v3.0-RC2/vehicle_status/required_vehicle_type_id.js:2]: Slightly better than "r", but if it means "result", can we just name it "result"? Or I would even argue that "partial" is a good variable name here.

**Review by testower — CHANGES_REQUESTED** [review]
Perhaps on the pedantic side, but I'd really like to change this trend of single letter and abbreviated variable names.

**Review by tdelmas — APPROVED** [review]

File: gbfs-validator/versions/partials/v3.0-RC2/vehicle_status/required_vehicle_type_id.js (hunk 2 above)
Comment (richfab) [reply gbfs-validator/versions/partials/v3.0-RC2/vehicle_status/required_vehicle_type_id.js]: Thank you for the suggestion! I renamed the variable to `partial`.

File: gbfs-validator/versions/partials/v3.0-RC2/system_information/required_store_uri.js (hunk 1 above)
Comment (richfab) [reply gbfs-validator/versions/partials/v3.0-RC2/system_information/required_store_uri.js]: Thank you for the suggestion! I renamed the variable to `partial`.

**Review by josee-sabourin — APPROVED** [review]

**Pull request merged**